proto/xudrpc.proto

```diff
@@ -136,13 +134,15 @@ message ExecuteSwapResponse {
 
 message GetInfoRequest {}
 message GetInfoResponse {
-  int32 num_peers = 1 [json_name = "num_peers"];
-  int32 num_pairs = 2 [json_name = "num_pairs"];
-  string version = 3 [json_name = "version"];
-  OrdersCount orders = 4 [json_name = "orders"];
-  LndInfo lndbtc = 5 [json_name = "lndbtc"];
-  LndInfo lndltc = 6 [json_name = "lndltc"];
-  RaidenInfo raiden = 7 [json_name = "raiden"];
+  string version = 1 [json_name = "version"];
+  string uri = 2 [json_name = "uri"];
+  repeated Address addresses = 3 [json_name = "addresses"];
```

Comment:
I think this would be better if it were like this:

```proto
string node_pub_key = 2 [json_name = "node_pub_key"];
repeated string uris = 3 [json_name = "uris"];
```

My thinking is we only will have one node pub key, but we could have multiple uris. And rather than breaking the uri down into its parts, making it a string will be nice so it can be simply copied as is and plugged directly into the `connect` command which sensibly takes a string.

Obviously the Service.getInfo method would have to change if this were changed.